src/node_options.cc

```diff
@@ -89,6 +89,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "experimental ES Module support and caching modules",
             &EnvironmentOptions::experimental_modules,
             kAllowedInEnvironment);
+  AddOption("--experimental-policy",
+            "use the specified file as a "
+            "security policy",
+            &EnvironmentOptions::experimental_policy,
+            kAllowedInEnvironment);
```

Comment:
sadly, at this point options still have to be manually documented in `doc/node.1` and `doc/api/cli.md`…

Reply:
added, phrasing could be better but I couldn't think of anything.